build.gradle

```diff
@@ -4,6 +4,9 @@ buildscript {
     repositories {
         jcenter()
     }
+    repositories {
+        mavenCentral()
+    }
```

Comment:
Removed. I initially added it due to the instructions on integrating Facebook, but now I see it only needs to be in the app's gradle (which it already is), and not in the root gradle.